components/nexus-security/src/test/java/org/sonatype/nexus/security/user/EmptyRoleManagementTest.java

```diff
@@ -23,6 +23,7 @@
 
 import org.junit.Test;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
The old `NexusTestSupport` class defines its own `assertEquals` so the process of removing it involved importing the appropriate JUnit method (or switching to the equivalent in hamcrest)